### ISSUE TITLE
A question about the learning rate of pre-trained VGG

### ISSUE THREAD

**Issue by Sucran** [Andrew-Qibin/DSS#10]
You had created an amazing and large model based on pre-trained VGG model, but you set the same learning rate in the pre-trained models and almost all the following layers in your training. However, the scale of saliency datasets are much smaller than ImageNet dataset, and the results may suffered from overfitting problem. So, how did you solve this problem in you training ?

**Comment by houqb** [comment]
decrease your initial learning rate, find a largest one that makes your model converge. You can also normalize the loss by the count of valid values as done in the new-version Caffe. For the scale of datasets,
it affects subtly.

**Comment by Sucran** [comment]
@Andrew-Qibin Thank you so much~   ：）
